molgenis/molgenis-emx2
# feat: simplified menu config to fix issue with catalogue relative URLs

Motivation: 
* in catalogue app we had issue of relative menu giving errors
* you had to provide very strange url with "../" to make it work

Solution: 
* relative menu items are automatically prefixed with a baseURL
* so now menu item to "schema" simply is set via "schema" instead of "../schema/"
* can still use absolute href starting with "/" or "http://" or "../" (recommend NOT to use, is for backward compatibility)
* updated menu manager with a bit of labels and documentation to hopefully make it even easier to follow

To test:
* default menu has been updated, so when you create a new schema you DON'T have "../" items
* in settings->menu you can see how that works
* in catalogue app go to 'settings' and then 'menu' and strip leading "../" and trailing "/"
